access-mgmt/README.adoc

```diff
@@ -0,0 +1,473 @@
+* A single operation (such as `Write`) for a resource or all operations for a resource
+
+If two or more permission settings in an ACL match a request being made to the Kafka broker and at least one of the matching permissions specifies that the action is denied, then the request is denied.
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://kafka.apache.org/documentation/#security_authz[Authorization and ACLs^] in Kafka documentation
+
+=== Supported ACL permissions in {product}
+
+An ACL acts as a mapping of permitted operations on specified resources for a selected account or for all accounts in an organization. An account can be a user account or a service account. Operations correspond to one or more Kafka APIs or request types that are applicable for the specified resource.
+
+For example, a `Read` operation for a `Topic` resource corresponds to the `Fetch`, `OffsetCommit`, and `TxnOffsetCommit` Kafka requests. A `Write` operation for a `Topic` resource corresponds to the `Produce` and `AddPartitionsToTxn` Kafka requests.
+
+The following table lists the supported ACL permissions in {product}.
+
+NOTE: The resource identifiers `Starts with` and `Is` support the wildcard character `*` to denote any occurrences of the specified resource. For example, `Topic` = `is *` means any topics in a Kafka instance.
+
+.Supported ACL permissions
+[cols="25%,25%,25%,25%"]
+|===
+h|Resource type
+h|Resource identifier
+h|Access type
+h|Operations
+
+|`Consumer group`
+
+(For consumer group access to a resource)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Read`
+* `Delete`
+* `Describe`
+
+|`Topic`
+
+(For access to a topic)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Read`
+* `Write`
+* `Create`
+* `Delete`
+* `Alter`
+* `Alter configs`
+* `Describe`
+* `Describe configs`
+
+|`Transactional ID`
+
+(For producer access to a resource)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Write`
+* `Describe`
+
+|`Kafka instance`
+
+(For access to Kafka instance permissions in ACLs)
+|None
+a|
+* `Allow`
+* `Deny`
+a|
+* `Alter`
+* `Describe`
+|===
+
+[role="_additional-resources"]
+.Additional resources
+https://kafka.apache.org/documentation/#security_authz_primitives[Authorization Primitives^] in Kafka documentation
+
+[id="proc-setting-permissions_{context}"]
+== Setting account permissions in {product}
+
+[role="_abstract"]
+In {product}, you can create Access Control Lists (ACLs) in Kafka instances and set permissions for how user accounts or service accounts can interact with an instance and its resources. Only administrators can manage the level of access of other users or of service accounts created by other users for a Kafka instance. If you're managing only service accounts that you created for a Kafka instance that you created, you don't need administrator rights.
```

Comment:
Same as other comment regarding an instance owner's ability to set ACLs for any account.

Reply:
Done.